workflows/dev/test_existing_browser.py

```
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+"""
+Test workflow on existing browser session
+"""
+import asyncio
+from workflow_use.workflow.service import Workflow
+
+async def test_workflow_on_existing_browser():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)